docs/modules/ROOT/nav.adoc

```diff
@@ -41,6 +41,7 @@
 ** xref:management-center-persistence.adoc[Persistence]
 ** xref:management-center-jvm-args.adoc[JVM Arguments]
 ** xref:management-center-ldap.adoc[LDAP Security Provider]
+* xref:lite-members.adoc[Deploying Lite Members]
```

Comment:
Could you move this section under `scaling-upgrading` section?

Reply:
Why? I don't think lite member deployment is related to scaling or upgrading. It should be a separate section.

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/248/commits/6327e0501175a31f219a06fd31d89146a76bc56f